presidio_cli/cli.py

```diff
@@ -118,8 +118,11 @@ def find_files_recursively(items, conf):
             for root, dirnames, filenames in os.walk(item):
                 for f in filenames:
                     filepath = os.path.join(root, f)
-                    if conf.is_text_file(filepath):
-                        yield filepath
+                    try:
+                        if conf.is_text_file(filepath):
+                            yield filepath
+                    except UnicodeDecodeError:
+                        pass  # Found non-text data
```

Comment:
Maybe there should be a log for this exception?
```suggestion
                    except UnicodeDecodeError as ex:
                        print(f'Presidio config is not a text file. {ex}')
```

Reply:
We don't want this printed anywhere, since it'll interfere with the actual output. In another iteration, we'll introduce a "verbose" mode which will print logs when the user asks for it.